app/controllers/discovered_hosts_controller.rb

```diff
@@ -25,8 +26,19 @@ def index
 
   def show
     # filter graph time range
+    @general =    {:Highlights => [ "productname", "memorysize", "interface", "manufacturer", "architecture", "macaddress", "processorcount", "physicalprocessorcount", "discovery_boot"]}
+    @categories = {:Storage => ["blockdevice"],
+                   :Hardware => ["hardw", "manufacturer", "memo", "process"],
+                   :Software => ["bios", "os", "discovery"]}
```

Comment:
In the regexp try to follow the pattern of `^(prefix1|prefix2)` but in case you want to compare by suffixes you can of course do something like `((suffix1|suffix2)$)|(^(prefix1|prefix2))`